package.json

```diff
@@ -37,6 +38,7 @@
   "homepage": "https://github.com/PalisadoesFoundation/talawa-api#readme",
   "dependencies": {
     "@graphql-tools/schema": "^8.4.0",
+    "@types/yargs": "^17.0.24",
```

Comment:
@EshaanAgg 

I think you haven't read the issue statement properly. This script not only needs to be executed in the dev environment, but required in production also.

It will be used by the CI/CD pipeline to execute and parse the arguments.

Reply:
Got it.